scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryTypes.scala

```diff
@@ -68,14 +68,23 @@ sealed trait Table extends Source {
   def spec: String
 
   def ref: GTableReference
+
+  def latest(bg: BigQuery): Table
```

Comment:
can we add `def latest()` as well?

Reply:
Also some scaladoc as we did for `Query`?

Reply:
Indeed, sorry I forgot that one :)

Do you know where I can see some examples how how to use this `Table` type or what is the most common way to use it? Then I can perhaps write a little scala doc here about it as I saw that
https://github.com/spotify/scio/issues/3208

Reply:
You can have a look at `scio-examples` as we have some things there like https://github.com/spotify/scio/blob/4ca965e0c1d82f8aafb2628eaa92edb5b5f64337/scio-examples/src/main/scala/com/spotify/scio/examples/cookbook/BigQueryTornadoes.scala#L34

The migration guide also gives you an overview of where it's used https://spotify.github.io/scio/migrations/v0.9.0-Migration-Guide.html#bigquery.

Although for this PR is enough just to add doc around the latest.